mypy.ini

```diff
@@ -48,6 +48,9 @@ ignore_missing_imports = True
 [mypy-test_torch]
 check_untyped_defs = False
 
+[mypy-torch._torch_docs]
+ignore_errors = True
```

Comment:
This seems like an unrelated fix, that'll probably get fixed separately because CI is broken now. Usually best to put that in a separate PR if you notice an unrelated problem, so it can get landed asap.